cowprotocol/cowswap
# feat(twap): twap review order - pt1

# Summary

This PR is based on and supersedes https://github.com/cowprotocol/cowswap/pull/2593
Implements part of https://github.com/cowprotocol/cowswap/issues/2577

Basic TWAP review details

![image](https://github.com/cowprotocol/cowswap/assets/43217/ec2f3a88-41dc-4823-8a9d-57e6942b27af)

- Price
- Slippage
- Limit price
- Minimum received

# To Test

1. On a Gnosis Safe with the handler already updated, go to advanced
2. Fill in the form
3. Click on review
* You should see the fields above properly filled and calculated

# Notes

1. Styling definitely not ready
2. Not yet in the structure proposed on https://github.com/cowprotocol/cowswap/issues/2577, will be the next PR
5. Remaining parts will come in subsequent PRs
